tasks/virtualenv.yml

```diff
@@ -33,7 +33,7 @@
   - name: setuptools>44.0.0,<45.0.0
     state: present
   - name: pyOpenSSL
-  - name: boto
+  - name: boto3
```

Comment:
RHEL/CentOS 7 won't have Python 3 installed and boto3 now only supports Python 3. Could you please pin to version `1.17.112` when using RHEL/CentOS 7?